src/webviews/home/homeWebview.ts

```diff
@@ -179,6 +180,26 @@ export class HomeWebviewProvider implements WebviewProvider<State, State, HomeWe
 
 	registerCommands(): Disposable[] {
 		return [
+			registerCommand(
+				`${this.host.id}.pull`,
+				() => {
+					void executeGitCommand({
+						command: 'pull',
+						state: {},
+					});
+				},
+				this,
+			),
+			registerCommand(
+				`${this.host.id}.push`,
+				args => {
+					void executeGitCommand({
+						command: 'push',
+						state: { flags: args.force ? ['--force'] : undefined },
+					});
+				},
+				this,
+			),
```

Comment:
Use `registerWebviewCommand` instead of `registerCommand` and make sure to pass the relevant refs (repo, branch, etc).

Reply:
Ignore the comment on converting `registerCommand` to `registerWebviewCommand`. I got them mixed up, sorry.

But do make sure you're passing the relevant refs (repo, branch, etc).